packages/sdk/src/protocols/base_protocol.ts

```diff
@@ -14,6 +18,7 @@ const DEFAULT_NUM_PEERS_TO_USE = 3;
 const DEFAULT_MAINTAIN_PEERS_INTERVAL = 30_000;
 
 export class BaseProtocolSDK implements IBaseProtocolSDK {
+  private healthManager: IHealthManager;
```

Comment:
we should understand if we want to support `waku.lightpush.health` as well as `waku.health` API

I don't have any preferences here